src/Context.js

```diff
@@ -44,3 +32,5 @@ export default class Context {
     return this.endpoints || {};
   }
 }
+
+export default new Context()
```

Comment:
`export` statements should end with semicolon: <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export>